### DISCUSSION TITLE
i want to decompile my  Bytecode but when i run this tool it show Unknown magic number 8224 in <file_name>

### DISCUSSION THREAD

**Discussion by Piyushsantoshpandey** [rocky/python-uncompyle6#490] [Q&A]
i want to decompile my  Bytecode but when i run this tool it show Unknown magic number 8224 in <file_name>

**Comment by rocky** [comment]
> i want to decompile my Bytecode but when i run this tool it show Unknown magic number 8224 in <file_name>

Why then, do you believe this is Python bytecode?
